### ISSUE TITLE
Is there a way to install pytorch3d in a virtual environment on Windows without using conda?

### ISSUE THREAD

**Issue by turugi-ni** [facebookresearch/pytorch3d#1837]
I need pytorch3d.structures as a module to run Unique3D on Windows.
Is there a way to install PyTorch3D in a venv virtual environment without using conda?

**Comment by bottler** [comment]
Yes, you don't need conda and a virtualenv should be fine. You just need to be careful to get the versions of cuda/cudatoolkit (if you are using GPU) and pytorch consistent with what your machine is running, and build PyTorch3D from source as described in other issues. If you need cuda, note that building on Windows is anyway quite complicated.
